sample/ApplicationInsightsHostingStartupSample/Views/Home/ScriptCheck.cshtml

```diff
@@ -0,0 +1,108 @@
+﻿@{
+    ViewData["Title"] = "Home Page";
+}
+
+<div id="myCarousel" class="carousel slide" data-ride="carousel" data-interval="6000">
+    <ol class="carousel-indicators">
```

Comment:
Can probably simplify some of this huge HTML (here and in the layout page). I think most of the content here won't work anyway because it required Bootstrap etc.